packages/vscode-extension/src/error/common.ts

```diff
@@ -95,6 +125,9 @@ export async function showError(e: UserError | SystemError) {
     const buttons = recommendTestTool
       ? [runTestTool, issue, similarIssues]
       : [issue, similarIssues];
+    if (shouldRecommendTeamsAgent) {
```

Comment:
What's the UI effect of showing 3/4 buttons?

Reply:
If at most four buttons. user can hover on the button to view full text.
![image](https://github.com/user-attachments/assets/cdf89e9d-b825-4f05-919d-046fc41ab612)

Reply:
OK thanks! Also, I'd suggest discussing with PM/UX to decide the button number and the primary button to achieve better effect because the main purpose is to increase entry point to troubleshooting.

Reply:
just left a comment to UX :) Thanks for your suggestion.